object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -219,16 +219,50 @@ private void checkPseudoAssignToOwning(Node node) {
       }
     }
     if (!assignedToOwning) {
-      // check if @CalledMethods type of return satisfies the @MustCall obligation
-      AnnotationMirror cmAnno =
-          typeFactory.getAnnotatedType(tree).getAnnotationInHierarchy(typeFactory.top);
-      if (!calledMethodsSatisfyMustCall(mustCallVal, cmAnno)) {
-        checker.reportError(
-            tree,
-            "required.method.not.called",
-            MustCallInvokedChecker.formatMissingMustCallMethods(mustCallVal),
-            TreeUtils.typeOf(tree).toString(),
-            "never assigned to an @Owning location");
+      if (typeFactory.biMap.inverse().containsKey(node)) {
```

Comment:
This code definitely needs some comments; I can't immediately see what it is doing.  Also, I was imagining that this whole `checkPseudoAssignToOwning` method would go away once temporary variables are properly implemented.  Do you agree?